primus_vk.cpp

```diff
@@ -1066,6 +1069,15 @@ void ImageWorker::copyImageData(uint32_t index, std::vector<VkSemaphore> sems){
       throw std::runtime_error("Layouts don't match at all");
     }
     TRACE_PROFILING_EVENT(index, "memcpy start");
+    VkMappedMemoryRange rendered_range {
+      VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE,
+      VK_NULL_HANDLE,
+      render_copy_image->mem,
+      0,
+      VK_WHOLE_SIZE
+    };
```

Comment:
You still didn't use named struct values:

```suggestion
    VkMappedMemoryRange rendered_range {
      .sType = VK_STRUCTURE_TYPE_MAPPED_MEMORY_RANGE,
      .pNext = VK_NULL_HANDLE,
      .memory = render_copy_image->mem,
      .offset = 0,
      .size = VK_WHOLE_SIZE
    };
```
please also adjust the other struct.